components/brave_wallet/browser/solana_tx_manager.cc

```diff
@@ -148,8 +148,10 @@ void SolanaTxManager::OnGetLatestBlockhashHardware(
   tx_state_manager_->AddOrUpdateTx(*meta);
 
   auto message_signers_pair = meta->tx()->GetSerializedMessage();
-  if (!message_signers_pair)
+  if (!message_signers_pair) {
     std::move(callback).Run(nullptr);
+    return;
```

Comment:
Same as above concerns - this is changing the behavior. This is fixing it IMO, but will result in callback not happening below anymore. Is this OK?

Reply:
Explained above.

Reply:
Marking as resolved; thanks for the replies on above

Reply:
It's a correct fix.